src/objects/Opportunity.object

```diff
@@ -832,6 +832,19 @@
         <trackTrending>false</trackTrending>
         <type>Lookup</type>
     </fields>
+    <fields>
+        <fullName>Batch_Number__c</fullName>
+        <description>The batch that created this Opportunity.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The batch that created this Opportunity.</inlineHelpText>
```

Comment:
@melissabarber have you had a chance to review help text and description for these fields?